src/navigators/HomeStack.tsx

```diff
@@ -6,12 +6,18 @@ import { AppTileScreen } from '../screens/AppTileScreen';
 import { CustomAppTileScreen } from '../screens/CustomAppTileScreen';
 import { TrackTileTrackerScreen } from '../screens/TrackTileTrackerScreen';
 import { AppNavHeader } from '../components/AppNavHeader';
+import { TrackerValuesContext } from '../components/TrackTile/main';
+import TrackTileSettingsScreen from '../screens/TrackTileSettingsScreen';
+import { t } from '@i18n';
 
 export type HomeStackParamList = {
   Home: undefined;
   'tiles/AppTile': { appTile: AppTile };
   'tiles/CustomAppTile': { appTile: AppTile };
   'tiles/TrackTile': { tracker: any; valuesContext: any };
+  'tiles/trackTileSettings': {
```

Comment:
The casing is different than the others. Do we want to make that consistent or are we switching to snack-case?

Reply:
Good catch.  Looking at this now, are we attempting to imbue semantic meaning into these route names?  Maybe we should have them align with their stack location.